### DISCUSSION TITLE
Welcome to enapso-graphdb-client Discussions!

### DISCUSSION THREAD

**Discussion by asheshgoplani** [innotrade/enapso-graphdb-client#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by anggerha** [comment]
![image](https://user-images.githubusercontent.com/42529312/199091239-ef1ee3f6-2399-45a2-b7f7-38b98b0e9255.png)

I got this error when installing. What should i do?

**Reply by Myasir92** [reply]
Can you please try again to install the package?

**Reply by anggerha** [reply]
It's installed now.. thanks @Myasir92😄
